### DISCUSSION TITLE
Loosened judgements for certain objects in standard

### DISCUSSION THREAD

**Discussion by ebolite** [ppy/osu#13146] [Ideas]
**Describe the new feature:** 
Certain objects in beatmaps would have a larger "Great" hit window.

The logic behind this feature is that it would reduce unfair judgements due to having no beat before the map begins and due to variable BPM. It would also help older beatmaps play as they were intended while still keeping slider head judgements.

**Proposal designs of the feature:** 
Objects within these categories would have their judgements loosened:

- The first two objects of any beatmap
- The first two objects after a break
- The first two objects after a BPM change whose factors do not contain the original BPM
- Slider heads

If an object falls into multiple categories, it would only have its judgement loosened once. For these purposes, slider tails would count as an object. Slider head judgements would be loosened to resemble legacy behavior more closely without being cheeseable.

_Note: This solution isn't perfect, as certain legacy maps ([such as Lan wings's Roze](https://osu.ppy.sh/beatmapsets/65994#osu/203906)) already account for their variable BPM by having lowered OD relative to their difficulty. Despite this, I still believe that a feature like this could go a long way in improving the perceived fairness of the game._

**Comment by smoogipoo** [comment]
We do have plans for per-hitobject difficulty adjustments, which is sort of touched upon in https://github.com/ppy/osu/issues/4060 . It's not going to be a "set in stone" thing that lazer does for you, it'll be at the mapper's discretion.

**Comment by ebolite** [comment]
In that case, maybe converted beatmaps should have their OD adjusted by default in order to closer resemble their previous behavior.
